files-generic/config/Config-gargoyle.in

```diff
@@ -75,9 +88,18 @@ menuconfig USE_GARGOYLE_PROFILE_PKGS
 			select PACKAGE_plugin-gargoyle-theme-light
 			select PACKAGE_plugin-gargoyle-theme-xeye
 			help
-				Plugins for alternative display themes
+			  Plugins for alternative display themes
 
 		menu "Extra Gargoyle package sets"
+			config GARGOYLE_EXTRA_UTILITIES
+				tristate "Utility packages"
+				default m
+				select PACKAGE_opkg
+				select PACKAGE_resolveip
+				select PACKAGE_swap-utils
```

Comment:
Wondering if swap-utils should be part of the usb package itself rather than any of these meta profiles? It's that plugin which utilises the swapon/swapoff capability